cache/src/main/kotlin/com/dropbox/android/external/cache4/RealCache.kt

```diff
@@ -33,7 +34,9 @@ internal class RealCache<Key : Any, Value : Any>(
     val expireAfterAccessDuration: Duration,
     val maxSize: Long,
     val concurrencyLevel: Int,
-    val clock: Clock
+    val clock: Clock,
+    val weigher: Weigher<Key, Value> = OneWeigher(),
+    val maxWeight: Long = UNSET_LONG
```

Comment:
Do we need the default parameters? this class is `internal` anyway so we can just choose the defaults in builder?

Reply:
Since the default builder is weigher agnostic it felt cleaner here.